tutorials/katacoda/thanos/2-lts/step2.md

```diff
@@ -79,7 +79,7 @@ docker run -d --net=host --rm \
     -v /root/prom-eu1:/prometheus \
     --name prometheus-0-eu1-sidecar \
     -u root \
-    quay.io/thanos/thanos:v0.24.0 \
+    quay.io/thanos/thanos:v0.25.0 \
```

Comment:
Do we need to update these to `0.25.1`?

Reply:
Good point, I think we should updated that in the `release-0.25` branch.

Since we've been also asked to include one more fix in a new minor version (https://cloud-native.slack.com/archives/CK5RSSC10/p1647361502703179), I might do that first and then update this PR to have everything in one go.